### DISCUSSION TITLE
v1.21.3

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#611] [Release]
**Changes:**
Simplify vscript stub function ([Launcher#627](https://github.com/R2Northstar/NorthstarLauncher/pull/627))

**Fixes:**
Fix spectating for LTS ([Mods#774](https://github.com/R2Northstar/NorthstarMods/pull/774))
Add validity check to `SaveStatsPeriodically_Threaded` ([Mods#769](https://github.com/R2Northstar/NorthstarMods/pull/769))
Add validity check to `HandleDistanceAndTimeStats_Threaded` to prevent crash ([Mods#767](https://github.com/R2Northstar/NorthstarMods/pull/767))

**Translation updates:**
Translations update from Weblate ([Mods#776](https://github.com/R2Northstar/NorthstarMods/pull/776))

**Contributors:**
@ASpoonPlaysGames @CTalvio @Khalmee @L1ghtman2k @harmony-weblate

<hr /><em>This discussion was created from the release <a href='https://github.com/R2Northstar/Northstar/releases/tag/v1.21.3'>v1.21.3</a>.</em>

**Comment by CTalvio** [comment]
Just commenting to confirm I've indeed had no issues with my changes, and the _stats crash has not recurred.
